### PR TITLE
I did some reordering and added a few

### PR DESCRIPTION
**Pull request by darrellshi**

### CODE REVIEW

File: Product Backlog.md [hunk 1]
```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
...
 7. As a user, I want to rate recipes, and view the public's ratings of recipes **(if time allows)**
-8. As a user, I want to find recipes based on items in my kitchen
 9. As a user, I want to be able to sort recipes by culinary style and genre **(if time allows)**
 10. As a user, I want to be able to find recipes based on prep time **(if time allows)**
-11. As a user, I want to be able to recover my password if I forget it or it becomes compromised
-12. As a user, I want to receive suggestions for recipes based on whats in my pantry and what I have liked in the past
 13. As a user, I want to plan my meals in advance to determine what ingredients I need **(if time allows)**
+14. As a user, I want to be reminded before food in my kitten expires and given suggested recipes to make use of the food **(if time allows)**
 14. Ad a user, I would like to post a picture of my completed recipe to social media **(if time allows)**
+15. As a developer, I would like to keep track of users' preferences and give more personalized recipe suggestions
```
Comment (jackhorton) [inline Product Backlog.md]: This should definitely be **(if time allows)**

File: Product Backlog.md [hunk 2]
```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
 ### Functional
 1. As a user, I want to create an account
 2. As a user, I want to log into my account
-3. As a user, I want to input and remove grocery items to my "kitchen/pantry"
+3. As a user, I want to be able to recover my password if I forget it or it becomes compromised
+4. As a user, I want to input grocery items to my "kitchen/pantry" by scanning barcodes
```
Comment (jackhorton) [inline Product Backlog.md]: I would also call this one **(if time allows)**

File: Product Backlog.md [hunk 3]
```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
...
+5. As a user, I want to input grocery items to my "kitchen/pantry" by typing item names
+6. As a user, I want to be able to remove grocery items
+7. As a user, I want to keep track of my current supply of grocery items
+8. As a user, I want to find recipes based on items in my kitchen
+9. As a user, I want to add allergies to limit the recipes I look for
```
Comment (jackhorton) [inline Product Backlog.md]: More (if time allows)

File: Product Backlog.md [hunk 4]
```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
...
 4. As a user, I want to know the nutritional information about a recipe **(if time allows)**
 5. As a user, I want to be able to save the recipes that I like
 6. As a user, I want to read the recipe easily
 7. As a user, I want to rate recipes, and view the public's ratings of recipes **(if time allows)**
-8. As a user, I want to find recipes based on items in my kitchen
 9. As a user, I want to be able to sort recipes by culinary style and genre **(if time allows)**
 10. As a user, I want to be able to find recipes based on prep time **(if time allows)**
-11. As a user, I want to be able to recover my password if I forget it or it becomes compromised
-12. As a user, I want to receive suggestions for recipes based on whats in my pantry and what I have liked in the past
```
Comment (jackhorton) [inline Product Backlog.md:24]: Why don't we like these two things?

Comment (darrellshi) [inline Product Backlog.md:24]: I moved line 11 to the front along with signup and login all those first time stuff.
And line 12 we already have "receive recipes based on what the user has". And user preferences I think it's more of a developer thing, so I added "as a developer". I can change it back if it makes more sense

File: Product Backlog.md [hunk 5]
```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
 ### Functional
 1. As a user, I want to create an account
 2. As a user, I want to log into my account
-3. As a user, I want to input and remove grocery items to my "kitchen/pantry"
```
Comment (darrellshi) [inline Product Backlog.md:15]: I split "input" and "remove" to be more detailed. On number 4, 5, 6

**Comment by jackhorton** [comment]
Looks good to me now

**Pull request merged**